src/geometry/euclidean/basic.lean

```diff
@@ -64,22 +64,23 @@ conveniently be developed in terms of vectors and then used to deduce
 corresponding results for Euclidean affine spaces.
 -/
 
-variables {V : Type*} [inner_product_space V]
+variables {V : Type*} [inner_product_space ℝ V]
```

Comment:
An update to the module doc string discussing how to declare a Euclidean affine space is needed.

Reply:
Done!